spread.yaml

```diff
@@ -16,17 +16,24 @@ backends:
                 kernel: GRUB 2
             - debian-unstable-64:
                 kernel: GRUB 2
+            - fedora-25-64
 
 path: /root/spread
 
 restore: |
-    apt autoremove -y
+    if [[ "$SPREAD_SYSTEM" = fedora-* ]]; then
```

Comment:
Can you please switch/case here. It it will look more natural and will do everything we need. We also should not have a default / else case as it will always be wrong, just be explicit.